api-specificatie/openapi.yaml

```diff
@@ -33,26 +42,15 @@ paths:
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/401"
         '403':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/403"
-        '409':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/409"
-        '410':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/410"
-        '415':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/415"
-        '429':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/429"
         '500':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/500"
-        '501':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/501"
-        '503':
```

Comment:
Waarom geen 503 Service unavailable? De service kan toch niet beschikbaar zijn (onderhoud)? Of zie je dit als teveel detail (valt onder default)

Reply:
Ik heb er even de API's voor het zaakgericht werken op nagekeken maar daar wordt nergens de 503 gedefinieerd. Dat hoeft natuurlijk geen argument te zijn het niet te doen. Zeker niet omdat in de API Designrules Extensions van de Nederlandse API strategie de 503 wel expliciet wordt benoemd. Maar, dat onderdeel is weer niet  normatief.

Reply:
Of we de 503 definiëren of niet, hij kan voorkomen (want is beter dan bij onbeschikbaarheid een 500 tonen). Vaak zal die niet door de applicatie worden geleverd, maar door een API gateway. Hetzelfde geldt voor een aantal situaties van 404 en 401 foutmeldingen. 
Dat verklaart misschien ook dat deze niet voor ZGW is gedefinieerd, ik vermoed dat daar de API specificaties vanuit de server implementatie is gegenereerd.

Reply:
besproken in overleg 503 moet wel worden opgenomen. De API definitie moet ook/primair bruikbaar zijn voor eindgebruikers, dus client applicaties. Zijn zullen meestal of altijd een intermediaire component zoals een API gateway ertussen hebben, die een 503 hoort te geven wanneer de API niet beschikbaar is